test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatMsbuildTestFixture.cs

```diff
@@ -9,7 +9,7 @@ namespace NuGet.XPlat.FuncTest
 {
     public class XPlatMsbuildTestFixture : IDisposable
     {
-        private readonly string _dotnetCli = DotnetCliUtil.GetDotnetCli();
+        internal readonly string _dotnetCli = DotnetCliUtil.GetDotnetCli();
```

Comment:
Why?
I don't see any other usages.

Reply:
Abood originally put the tests in NuGet.XPlat.FuncTest and needed this change. But since I moved the tests to Dotnet.Integration.Test, I forgot to undo this change.